### PR TITLE
Update covid form and move section forms to config-ses

### PR DESCRIPTION
**Pull request by MiguelAHPpih**

### CODE REVIEW

File: configuration/pih/htmlforms/section-history.xml [hunk 1]
```diff
@@ -0,0 +1,750 @@
...
+
+    <section id="patient-history" sectionTag="fieldset" headerTag="legend" headerStyle="title"
+             headerCode="zl.consultNote.patientHistory.title">
+        <div class="section-container-color">
+            <!-- ANTECEDENTS PERSONNELS / HABITUDES -->
+
+            <div class="two-columns">
+                <!-- Allergy is intentionally ommitted because it should be handled in more detail through the Allergy UI -->
+                <pastMedicalHistoryCheckbox concept="PIH:ASTHMA" label="pihcore.ncd.asthma"/>
+                <pastMedicalHistoryCheckbox concept="PIH:HEART DISEASE" label="pihcore.ncd.heartdisease" specify="true" class="medicalHistoryText patientHistory" />
```
Comment (MiguelAHPpih) [inline configuration/pih/htmlforms/section-history.xml:305]: Added labels for correct translation

File: configuration/pih/htmlforms/section-history.xml [hunk 2]
```diff
@@ -0,0 +1,750 @@
...
+            </div>
+        </section>
+    </includeIf>
+
+    <section id="family-history" sectionTag="fieldset" headerTag="legend" headerStyle="title"
+             headerCode="zl.consultNote.familyHistory.title">
+        <div class="section-container">
+            <!-- ANTECEDENTS HEREDO-COLLATERAUX -->
+            <familyHistoryRelativeCheckboxes concept="PIH:ASTHMA" relatives="PIH:FATHER,PIH:MOTHER,PIH:SIBLING,PIH:OTHER"/>
+            <familyHistoryRelativeCheckboxes concept="PIH:HEART DISEASE" label="pihcore.ncd.heartdisease" relatives="PIH:FATHER,PIH:MOTHER,PIH:SIBLING,PIH:OTHER"/>
```
Comment (MiguelAHPpih) [inline configuration/pih/htmlforms/section-history.xml:275]: Added labels for correct translation

File: configuration/pih/htmlforms/section-plan.xml [hunk 3]
```diff
@@ -0,0 +1,682 @@
...
+                    <template>
+                        <p>
+                            <obs conceptId="PIH:Lab test ordered coded"
+                                 answerConceptId="{labName}"
+                                 style="checkbox"/>
+                        </p>
+                    </template>
+
+                    <!-- Hematology -->
+                    <render labName="PIH:HEMOGLOBIN" codelabel="pihcore.lab.hemoglobin" comment="Hemoglobina"/>
```
Comment (MiguelAHPpih) [inline configuration/pih/htmlforms/section-plan.xml:219]: Added labels for correct translation

File: configuration/pih/htmlforms/section-plan.xml [hunk 4]
```diff
@@ -0,0 +1,682 @@
...
+                    <template>
+                        <p>
+                            <obs conceptId="PIH:Lab test ordered coded"
+                                 answerConceptId="{labName}"
+                                 style="checkbox"/>
+                        </p>
+                    </template>
+
+                    <!-- Serology -->
+                    <render labName="CIEL:163618" codelabel="pihcore.lab.Chlamydia" comment="Chlamydia"/>
```
Comment (MiguelAHPpih) [inline configuration/pih/htmlforms/section-plan.xml:317]: Added label for correct translation

File: configuration/pih/htmlforms/section-plan.xml [hunk 5]
```diff
@@ -0,0 +1,682 @@
...
+                    <template>
+                        <p>
+                            <obs conceptId="PIH:Lab test ordered coded"
+                                 answerConceptId="{labName}"
+                                 answerCode="{codeLabel}"
+                                 style="checkbox"/>
+                        </p>
+                    </template>
+
+                    <render labName="CIEL:163722" codeLabel="pihcore.lab.hivRapid" comment="HIV RT"/>
```
Comment (MiguelAHPpih) [inline configuration/pih/htmlforms/section-plan.xml:400]: Added label for correct translation

File: configuration/pih/htmlforms/section-plan.xml [hunk 6]
```diff
@@ -0,0 +1,682 @@
...
+        <div class="section-container-color">
+            <includeIf velocityTest="!$config.isComponentEnabled('radiology')">
+
+                <!-- ToDo: This is different from the Radiology app which uses encounter
+                     and order.  This uses  'Radiology procedure performed'.
+                     Add interpretation and date -->
+                <p>
+                    <obs conceptId="PIH:Radiology image ordered"
+                         answerConceptId="PIH:Chest 1 view (XRay)"
+                         codelabel="pihcore.radiology.Chest_oneview"
```
Comment (MiguelAHPpih) [inline configuration/pih/htmlforms/section-plan.xml:467]: Added label for correct translation

**Review by mogoodrich — APPROVED** [review]
LGTM!

**Pull request merged**